builtin/providers/aws/resource_aws_route53_record.go

```diff
@@ -230,10 +242,6 @@ func resourceAwsRoute53RecordRead(d *schema.ResourceData, meta interface{}) erro
 		StartRecordType: aws.String(d.Get("type").(string)),
 	}
 
-	if v, ok := d.GetOk("set_identifier"); ok {
-		lopts.StartRecordIdentifier = aws.String(v.(string))
-	}
-
```

Comment:
same question, when / why was this removed? Maybe you just need to update from `master`